src/Lifecycle.js

```diff
@@ -352,11 +352,14 @@ export function setLoggedIn(credentials) {
 async function _doSetLoggedIn(credentials, clearStorage) {
     credentials.guest = Boolean(credentials.guest);
 
+    const softLogout = isSoftLogout();
```

Comment:
won't `clearStorage` still be `true` here when `Login` finishes your re-login through `setLoggedIn`?

Reply:
This PR doesn't solve login. That's https://github.com/vector-im/riot-web/issues/10236